artworks/views.py

```diff
@@ -1,3 +1,64 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect, reverse
+from .forms import ArtForm
+from django.db import transaction
+from .models import Category, Artwork, ArtLike
+from django.http import HttpResponseRedirect
+from django.contrib.auth import authenticate
+
+
+def upload_view(request):
+    if request.method == "POST":
+        form = ArtForm(request.POST, request.FILES)
+        if form.is_valid():
+            with transaction.atomic():
+                form_save = form.save(commit=False)
+                tags = [tag.lower() for tag in form.cleaned_data['category'].replace(' ', '').split(',')]
```

Comment:
What about `[tag.strip().lower() for tag in form.cleaned_data['category'].split(',')]`? Is it easier to read?